executor/show.go

```diff
@@ -219,7 +214,7 @@ func (e *ShowExec) fetchShowProcessList() error {
 			pi.Host,
 			pi.DB,
 			pi.Command,
-			t,
+			uint64(time.Since(pi.Time) / time.Second),
```

Comment:
I'm afraid this is not a real fix.
pi.Time maybe uninitialized.... say, 1970.1.1....

Reply:
@mccxj Please address the comment here.